model/example_domain_test.go

```diff
@@ -0,0 +1,618 @@
+package model_test
+
+import (
+	"fmt"
+
+	"github.com/nextmv-io/sdk/model"
+)
+
+func ExampleNewDomain_empty() {
+	domain := model.NewDomain()
+
+	fmt.Println(domain.Empty())
+
```

Comment:
I will only make this comment once, but it applies to all tests. We should remove the newline before the `// Output:` declaration. This is because the godocs render a blank new line before the closing curly brace that we do not want. I'm copying the text from the rendered godoc so that you can visualize what I mean.
```suggestion
```

```go
package main

import (
	"fmt"

	"github.com/nextmv-io/sdk/model"
)

func main() {
	domain := model.NewDomain()

	fmt.Println(domain.Empty())

}
```